docs/_content/overige/technisch/zds-vs-zgw.md

```diff
@@ -7,27 +7,22 @@ De Zaak- en Documentservices (ZDS) wijken af van de Zaakgericht werken (ZGW)
+Betrokkenen en/of gerelateerde objecten kunnen geheel of gedeeltelijk, in ieder geval met matchinggegevens doorgegeven worden. Deze worden veelal in het zaaksysteem opgeslagen.	                                                             | een relatie naar een andere resource is een verwijzing naar die resource. Die resource wordt opgeslagen in de betreffende registratie. Dat kan een zaakregistratie (ZRC) zijn maar ook een andere (met een eigen API). 
+bestaan voornamelijk uit a-synchrone services	                                            | bestaan uit alleen maar synchrone services
+Een [GEMMA 1](https://www.gemmaonline.nl/index.php/Overzicht_GEMMA_1_referentiecomponenten) Zaaksysteem bevat ook (generieke) zaakafhandelfunctionaliteit. Deze hoeft niet per se via ZDS te communiceren met het eigen zakenmagazijn  | Een [GEMMA 2](https://www.gemmaonline.nl/index.php/Overzicht_generieke_referentiecomponenten_GEMMA_2) [Zaakregistratiecomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-a97b6545-d5a7-485d-9b13-3ce22db5b9cf) registreert alleen zaken en (verwijzingen naar) gerelateerden. Afhandelen gebeurt in een [zaakafhandelcomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-f2dfbd0b-9d36-405c-bdbe-827f3296de29) (ZAC). Communicatie vindt plaats via de ZGW API's
+Zaak- Documentservices is gemaakt om taakspecifieke applicaties te koppelen aan een zaaksysteem. | ZGW API's zijn gemaakt om zaakgericht werken te ondersteunen
+Zaak- Documentservices beschrijven alleen hoe zaken, documenten en besluiten in een zaaksysteem vastgelegd kunnen worden. Er wordt niet beschreven hoe een ZTC bevraagd moet worden etc.                                                           | ZGW API's ondersteunen alle onderdelen van het zaakgericht werken, ook het bevragen van een ZTC etc.
+De Zaak- Documentservices standaard bestaat uit een specificatiedocument, xsd schema's en documentatie van de onderliggende StUF en CMIS standaarden.                                                                                | ZGW API standaard bestaat uit een OAS 3 yaml bestand, functionele en technische documentatie,  referentie implementaties (voorbeeld en proof of concept)
+Beheer van de Zaak- Documentservices gebeurt in de werkgroep Beheer Zaak- Documentservices. Daarnaast volgt de standaard de ontwikkelingen in de StUF standaard (voor zover van toepassing).                                                      | Beheer van de standaard zal plaats vinden onder regie van [VNG-Realisatie](https://www.vngrealisatie.nl/). Belanghebbenden kunnen wensen en bevindingen of zelfs wijzigingsvoorstellen indienen via issues en pullrequests. 
+Zaak- Documentservices kent extraElementen, label-value combinaties voor gegevens die niet in het RGBZ 1 opgenomen zijn.  | ZGW API's werken op resources. Wanneer een attribuut of zelfs een resource niet bekend is moet deze toegevoegd worden aan de API die bij de juiste registratie hoort.
```

Comment:
Dit geeft volgens mij een verkeerd beeld door het zinsdeel:

> Wanneer een attribuut of zelfs een resource **niet bekend is** [...]

Het gaat er niet om dat het niet bekend is, maar dat we niet allerlei domein data bij een zaak kunnen opslaan.

Als we het kopje "Additionele informatie opslaan bij een Zaak" nemen, zou ik eerder aangeven:

ZDS biedt de mogelijkheid om middels `extraElementen` informatie die niet in de standaard is opgenomen, toch toe te voegen aan bepaalde objecten, zoals een zaak. In de ZGW APIs is er voor gekozen deze constructie niet mogelijk te maken om compatibiliteit te bewaren. Wel kent een zaak bijvoorbeeld `eigenschappen` die in het ZTC zijn gedefinieerd op het zaaktype en in een zaak van een waarde kunnen worden voorzien. Als er zoveel extra informatie is dat `eigenschappen` niet werkbaar is, dan kan er altijd een nieuwe registratie API worden gerealiseerd en naar deze API worden verwezen middels een `zaakobject` (bijvoorbeeld een wegdeel), of vanuit de nieuwe API naar een zaak (bijvoorbeeld een melding).

Reply:
Domeindata zou nog kunnen wanneer dit uitgewerkt is. Het principe achter extraElementen en aanvullendeElementen is echter dat in een koppeling gegevens gebruikt kunnen worden die helemaal niet in het informatiemodel of de StUF berichten beschreven is. Bijvoorbeeld proprietary velden of een eigen enumeratie van een bestaand veld oid. 
Eigenschappen zijn netjes beschreven in de ZTC, dat is mooi maar dat zijn wel eigenschappen op zaaktype-niveau. Die gegevens zijn controleerbaar omdat ze uitgewerkt staan in de ZTC.
Een eigen key-value veld is echt maatwerk. Bovendien is er de spelregel dat wanneer een ontvanger het gegeven niet kent (de key komt niet in zijn lijstje voor) dat dit gegeven dan genegeerd kan worden. Dit maakt het makkelijker on the fly gegevens toe te voegen maar het wordt er niet overzichtelijker van.

Reply:
Let op: inderdaad is er de mogelijkheid om domeinspecifieke gegevens als eigenschappen bij een zaaktype te definieren. Echter: vaak is de behoefte aan dit soort gegevens een aanwijziging voor een objecttype dat in een eigen registratie moet worden ondergebracht. In mijn visie moeten eigenschappen zo min mogelijk worden gebruikt en moet er worden gestreefd naar aparte registraties voor entiteiten buiten ZGW.